skyllh/core/pdf.py

```diff
@@ -675,26 +691,31 @@ def __init__(self, axis_binnings, pdf_path_to_splinetable=None, pdf_grid_data=No
                 vmax=axis_binning.upper_edge
             ))
 
-       
-        pdf_loaded = False
+        # need either splinetable or value grid
+        if((pdf_path_to_splinetable is None) and (pdf_grid_data is None)):
```

Comment:
better: path_to_pdf_splinetable

Reply:
resolved in https://github.com/IceCubeOpenSource/skyllh/pull/22/commits/d33951c1cff6af86a1771b651c729295daee7fde